server/testing/src/main/java/org/apache/james/util/docker/Images.java

```diff
@@ -22,10 +22,8 @@
 public interface Images {
     String FAKE_SMTP = "weave/rest-smtp-sink:latest";
     String RABBITMQ = "rabbitmq:3.9.18-management";
-    String ELASTICSEARCH_2 = "elasticsearch:2.4.6";
     String ELASTICSEARCH_6 = "docker.elastic.co/elasticsearch/elasticsearch:6.3.2";
     String ELASTICSEARCH_7 = "docker.elastic.co/elasticsearch/elasticsearch:7.10.2";
-    String NGINX = "nginx:1.22";
```

Comment:
Why deleting this?

Reply:
It uses only for quick test containers library, and nowhere re-use it. Remove it will be less noise